bool door_open
bool door_close
bool pole_out
bool pole_in
bool charge_ready
bool ac220_charging